lib/src/components/NxCheckbox/NxCheckbox.scss

```
@@ -14,38 +14,76 @@
   @include regular();
   @include focusable-toggle-input(".nx-checkbox__box");
 
-  cursor: pointer;
-  display: inline;
-  font-size: 14px;
-  line-height: 20px;
-  overflow: hidden;
+  display: block;
+  margin-bottom: $nx-spacing-xs;
   white-space: nowrap;
+  max-width: max-content;
+
+  &:hover {
+    cursor: pointer;
+
+    .nx-checkbox__box {
+      border-color: #2c2c2c;
+      filter: none;
+    }
+  }
 }
 
 .nx-checkbox__box {
-  border: $nx-border-darker;
+  background-color: #fff;
+  border: 1px solid #d1d6f0;
   border-radius: 2px;
+  box-sizing: border-box;
   display: inline-block;
-  height: 14px;
-  margin-right: 8px;
+  height: 16px;
+  margin-right: $nx-spacing-xs;
   position: relative;
-  vertical-align: -2px;
-  width: 14px;
+  vertical-align: 1px;
+  width: 16px;
 
   .fa-check {
-    margin-bottom: 1.5px;
+    color: #fff;
+    font-size: $nx-font-size-s;
+    margin-bottom: 6px;
```

Since you're doing other vertical tweaks using `vertical-align`, you could stick to that here too.  This looks right to me:
```suggestion
    vertical-align: 0.28em;
```
Or `vertical-align: 4px;`, if you prefer

---

> Since you're doing other vertical tweaks using `vertical-align`, you could stick to that here too. This looks right to me:
> 
> Or `vertical-align: 4px;`, if you prefer

I changed it to to px just to keep the units consistent. https://github.com/sonatype/sonatype-react-shared-components/pull/117/commits/b3678a23321a00d8c3e830869c24ef2e17ad71a9